indico/modules/events/registration/forms.py

```diff
@@ -423,3 +424,33 @@ class RejectRegistrantsForm(IndicoForm):
 
     def is_submitted(self):
         return super().is_submitted() and 'submitted' in request.form
+
+
+class RegistrationTagForm(IndicoForm):
+    """Form to create a new registration tag."""
+
+    title = StringField(_('Title'), [DataRequired()])
+    color = StringField(_('Color'), [DataRequired()], widget=ColorInput())
```

Comment:
Could we use the existing color picker we have? Either `ColorPickerWidget` for free-form color choice (but that may be a bit problematic due to the text color being always white) or `IndicoSinglePalettePickerField` to select colors from a palette. We could use `ColorMixin` in the model and then `IndicoPalettePickerField` as well; that's the color palette which we also use e.g. for sessions (those entries all have fg/bg color pairs that work well together).